politeiawww/config.go

```diff
@@ -482,6 +492,24 @@ func loadConfig() (*config, []string, error) {
 
 	// Verify mode
 	switch cfg.Mode {
+	case cmsWWWMode:
+		cfg.Mode = cmsWWWMode
+		cfg.MailAddress = defaultCMSMailAddress
+		switch {
+		case cfg.CmsHost == "":
+			return nil, nil, fmt.Errorf("the cmshost param is required, while in cmswww mode")
+		case cfg.CmsRootCert == "":
+			return nil, nil, fmt.Errorf("the cmsrootcert param is required, while in cmswww mode")
+		case cfg.CmsCert == "":
+			return nil, nil, fmt.Errorf("the cmscert param is required, while in cmswww mode")
+		case cfg.CmsKey == "":
+			return nil, nil, fmt.Errorf("the cmskey param is required, while in cmswww mode")
+		}
+
+		cfg.CmsRootCert = cleanAndExpandPath(cfg.CmsRootCert)
+		cfg.CmsCert = cleanAndExpandPath(cfg.CmsCert)
+		cfg.CmsKey = cleanAndExpandPath(cfg.CmsKey)
+
 	case politeiaWWWMode:
```

Comment:
The validation of the cache params should be moved inside of the `politeiaWWWMode` case so that they don't get validated when running in cms mode.

It also looks like politeiawww still tries to connect to the cache in `_main()` even when running in cms mode.  You probably want to turn off the cache cockroachdb instance and then run politeiawww in cms mode to see what errors out.

Reply:
Well I believe we'll still be using the cache db for comments and such.